src/Alpha_complex/include/gudhi/Alpha_complex_3d.h

```diff
@@ -554,8 +554,10 @@ Weighted_alpha_complex_3d::Weighted_point_3 wp0(Weighted_alpha_complex_3d::Bare_
     std::clog << "cells \t\t" << count_cells << std::endl;
 #endif  // DEBUG_TRACES
     // --------------------------------------------------------------------------------------------
-    // As Alpha value is an approximation, we have to make filtration non decreasing while increasing the dimension
-    complex.make_filtration_non_decreasing();
+    if (Complexity == complexity::FAST)
```

Comment:
According to the benchmark referred to in #57, the results with 'safe' mode in 3D are different with some tested data. Shouldn't it be just the 'exact' mode where the make_filtration_non_decreasing may be removed safely like in dD?

Reply:
@Hind-M Which one ? For 3D, you have to look algo `alpha 3D *`, but not for `alpha dD *`. Maybe this big table is confusing

Reply:
Oh ok I see, I didn't notice that the algo can be different in 3D table aside from the mode. Maybe the two given tables should be split by algo and not by type of data, or the second table may have two sub-tables (algo alpha3D and algo alpha dD ?). Anyway that's how I read it considering that 4D data was an example used with alpha dD and 3D data with alpha 3D, I don't know if people read it that way too ...